plugin/storage/factory_config.go

```diff
@@ -69,11 +78,22 @@ func FactoryConfigFromEnvAndCLI(args []string, log io.Writer) FactoryConfig {
 	if depStorageType == "" {
 		depStorageType = spanWriterTypes[0]
 	}
+	ratioEnvVariable := os.Getenv(DownSamplingRatio)
+	// Default ratio to drop spans is 0
+	downSamplingRatio, err := strconv.ParseFloat(ratioEnvVariable, 64)
+	if err != nil {
```

Comment:
we should log the error

Reply:
I don't see a logger being passed here, is there an example for how we do that?

Reply:
look above on L71